apps/system/fxa/test/marionette/fxa_screen_flow_test.js

```diff
@@ -14,19 +13,24 @@ marionette('Firefox Accounts Screen Flow Test (UITest app)', function() {
       client = marionette.client({
         profile: {
           prefs: {
+            'focusmanager.testmode': true,
             'identity.fxaccounts.auth.uri': 'http://' +
               config.SERVER_HOST + ':' +
               config.SERVER_PORT + '/' +
               config.SERVER_PATH
-            }
+          }
         }
       });
 
-  setup(function() {
-    fxaUser= new FxAUser();
+  setup(function(done) {
+    fxaUser = new FxAUser();
     app = new FxA(client);
     Server.create(FxA.SERVER_ARGS, function (err, _server) {
+      if (err) {
+        console.error(err);
```

Comment:
Do we need to continue the test even if the server is not created? If not, we could add below:
```js
throw err;
```